plugins/chat/app/services/chat/auto_leave_channels.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
```

Comment:
Even if tested by other means, we at least need one test of the happy path here

Reply:
Ideally I would do the opposite, happy path of the boundaries, but full test coverage of the service